python-sdk/nuscenes/predict/helper.py

```diff
@@ -0,0 +1,181 @@
+
+from typing import Dict, Tuple, Any, List
+from pyquaternion import Quaternion
+from nuscenes import NuScenes
+from nuscenes.utils.geometry_utils import transform_matrix
+from nuscenes.eval.common.utils import quaternion_yaw
+import numpy as np
+
+MICROSECONDS_PER_SECOND = 1e6
+BUFFER = 0.15 # seconds
+
+def angle_of_rotation(yaw: float) -> float:
```

Comment:
@freddyaboulton Maybe a note on the conventions we are converting between?

Reply:
Done